fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsManager.java

```diff
@@ -133,7 +139,25 @@ private Table validateTableName(TableName dbTableName) throws AnalysisException
         String tableName = dbTableName.getTbl();
 
         Database db = Catalog.getCurrentCatalog().getDbOrAnalysisException(dbName);
-        Table table = db.getTableOrAnalysisException(tableName);
-        return table;
+        return db.getTableOrAnalysisException(tableName);
+    }
+
+    public void alterTableStatistics(StatisticsTaskResult taskResult) throws AnalysisException {
+        StatsGranularityDesc desc = taskResult.getGranularityDesc();
+        long tableId = desc.getTableId();
+        Map<String, String> statsNameToValue = taskResult.getStatsNameToValue();
+        this.statistics.updateTableStats(tableId, statsNameToValue);
```

Comment:
should we drop table's statistics if we dropped table?
if that, could be a problem that we don't check whether table exists?

Reply:
Yes, I have already checked the relevant information before updating the statistics, you can refer to my latest submitted code. We have not implemented the cleanup statistics and automatic update statistics in this issue, and will be implemented in the next issue.